app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js

```diff
@@ -28,26 +29,17 @@ describe("<FieldTranslationRow />", () => {
     }
   });
 
-  beforeEach(() => {
-    ({ component } = setupMockFormComponent(FieldTranslationRow, {
-      props: { field: fromJS(field1), selectedLocaleId: "fr" },
-      state
-    }));
-  });
+  const props = { field: fromJS(field1), selectedLocaleId: "en" };
 
   it("should render <FieldTranslationRow />", () => {
-    expect(component.find(FieldTranslationRow)).to.have.lengthOf(1);
+    localesToRender.mockReturnValue([fromJS({ id: "en" })]);
+    mountedFormComponent(<FieldTranslationRow {...props} />, state);
+    expect(screen.getByText("forms.manage")).toBeInTheDocument();
   });
 
   it("should render the <FormSectionField /> for the available languages", () => {
```

Comment:
This test should test if there is a form section field for en, ar, and fr locales. There should be 3 inputs with different name attributes
```
      "fields.field_1.display_name.en",
      "fields.field_1.display_name.fr",
      "fields.field_1.display_name.ar"

```

Reply:
Looks like the return of localesToRender is returning an plain array. You can remove the fromJS

Reply:
Done